cl/simple_pages/templates/help/recap_email_help.html

```diff
@@ -60,13 +60,13 @@ <h2>Why is this useful to the public?</h2>
 
 
     <h2>How and why is this free? I trust things I pay&nbsp;for.</h2>
-    <p>We're <a href="{% url "donate" %}?referrer=recap-dot-email">happy to accept donations from those using this service</a>, but the reason @recap.email is free is that we believe open legal data is key to a functioning democracy. When you use this system, you contribute to a better country. That's why this service is free.
+    <p>We're <a href="https://donate.free.law/forms/membership">happy to accept donations from those using this service</a>, but the reason @recap.email is free is that we believe open legal data is key to a functioning democracy. When you use this system, you contribute to a better country. That's why this service is free.
     </p>
     <p>We're able to make this free for individual users by charging organizations for specialized services, like the webhooks described below.
     </p>
 
     <h2>What's the best way to simply contribute to the&nbsp;cause?</h2>
-    <p>Well, we always need <a href="{% url "donate" %}">financial support</a> so we can build things like this, but if you just want to contribute your documents to the RECAP Archive, simply add <code>archive@recap.email</code> as a secondary recipient in your CM/ECF account. Once added, we'll process your notification emails in realtime, without linking them to any CourtListener account, sending you any emails, etc.
+    <p>Well, we always need <a href="https://donate.free.law/forms/membership">financial support</a> so we can build things like this, but if you just want to contribute your documents to the RECAP Archive, simply add <code>archive@recap.email</code> as a secondary recipient in your CM/ECF account. Once added, we'll process your notification emails in realtime, without linking them to any CourtListener account, sending you any emails, etc.
```

Comment:
I'm sorry, I think both of these should be `supportflp` links. :) 

I know, I'm at 34 comments and it's too many!